src/schema/v2/bank_account.ts

```diff
@@ -0,0 +1,43 @@
+import {
+  GraphQLObjectType,
+  GraphQLString,
+  GraphQLNonNull,
+  GraphQLFieldConfig,
+} from "graphql"
+import { InternalIDFields } from "schema/v2/object_identification"
+import { ResolverContext } from "types/graphql"
+
+// fields: https://github.com/artsy/gravity/blob/main/db/schema.rb
+export const BankAccountType = new GraphQLObjectType<any, ResolverContext>({
+  name: "BankAccount",
+  fields: () => ({
+    ...InternalIDFields,
+    bankName: {
+      type: new GraphQLNonNull(GraphQLString),
+      description: "Bank name",
+      resolve: ({ bank_name }) => bank_name,
+    },
+    accountHolderName: {
+      type: GraphQLString,
+      description: "Name on the bank account",
+      resolve: ({ account_holder_name }) => account_holder_name,
+    },
+    last4: {
+      type: new GraphQLNonNull(GraphQLString),
+      description: "Last four characters of the account identifier",
+    },
+  }),
+})
+
+export const BankAccount: GraphQLFieldConfig<void, ResolverContext> = {
+  type: BankAccountType,
+  description: "A user's bank account",
+  args: {
+    id: {
+      type: new GraphQLNonNull(GraphQLString),
+      description: "The ID of the bank account",
+    },
+  },
+  resolve: (_root, { id }, { bankAccountLoader }) =>
```

Comment:
This is nitpicky, but it's always good to avoid implicit returns unless it's like `(x, y) => x + y`. I suspect this resolver will in the future need to be inspected somehow, which would require unwinding the implicit return to add a console log or more code or a `debugger` statement.

Reply:
ah yeah 100 percent agree